gui/src/app/components/Splitter.tsx

```diff
@@ -0,0 +1,60 @@
+import DevbookSplit, {
```

Comment:
I know there's a cite later on in the file, but it might be nice to note up top that this is taken in whole cloth to fix an outstanding regression in the library.